core-web/apps/dotcms-ui/src/app/api/services/guards/ema-app/edit-page.guard.spec.ts

```diff
@@ -103,7 +103,7 @@ describe('EditPageGuard', () => {
         result.subscribe(() => {
             expect(router.navigate).toHaveBeenCalledWith(['edit-ema'], {
                 queryParams: {
-                    url: 'some-url',
+                    url: '/some-url',
```

Comment:
We should take boths... I think meaning like the `queryParam.url` can come with and without `/` and we should work with both.